lighthouse-core/report/v2/renderer/dom.js

```diff
@@ -56,7 +55,7 @@ class DOM {
    * @throws {Error}
    */
   cloneTemplate(selector, context) {
-    const template = context.querySelector(selector);
+    const template = /** @type {HTMLTemplateElement} */ (context.querySelector(selector));
```

Comment:
interesting! The devtools compiler must have an extra check for explicit nullability?